projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { INotebookTracker } from '@jupyterlab/notebook';
+import { IThemeManager } from '@jupyterlab/apputils';
+
+const addNumberElement = (number: Number, node: Element) => {
+  const numberElement = document.createElement('div');
+  numberElement.innerText = String(number);
+  numberElement.classList.add('jp-vdk-cell-num');
```

Comment:
Why does the numbering start from 0? 😊

Reply:
@ivakoleva do you think I should change it to start from 1?

Reply:
I think we may match ones line numbering of an IDE editor, starting from 1

Reply:
changed